src/web/mjs/connectors/ReaperScansFR.mjs

```diff
@@ -1,13 +1,17 @@
-import WordPressMangastream from './templates/WordPressMangastream.mjs';
+import WordPressMadara from './templates/WordPressMadara.mjs';
 
-export default class ReaperScansFR extends WordPressMangastream {
+export default class ReaperScans extends WordPressMadara {
```

Comment:
do not change the class name, should be the same as the id and correlated to the name/label.